src/components/Modals/GuideModal/GuideModal.scss

```diff
@@ -0,0 +1,3 @@
+.guide-modal {
+    width: 44rem;
+}
```

Comment:
tbh you dont need this scss file, we can just write it using tailwind

Reply:
this modal will be replaced by tooltip instead